materials/ggplot.md

```diff
@@ -65,110 +68,124 @@ ggplot(dm_ts_data, aes(x = year, y = count)) +
 
 ### Layers
 
 * Usage
-	* `ggplot()` sets defaults for layers
-	* Combine layers with `ggplot()` using `+`
-	* Must have at least one layer to plot
-	* Add additional layers, as necessary
-		* Order matters
+    * `ggplot()` sets defaults for layers
+    * Combine layers with `ggplot()` using `+`
+    * Must have at least one layer to plot
+    * Add additional layers, as necessary
+        * Order matters
 
 * Combining different kinds of layers
 
 ```
-ggplot(ts_data, aes(x = year, y = count)) +
-  geom_line() +
+ant_acacia <- filter(acacia, ANT %in% c("CM", "CS", "TP"))
+ggplot(ant_acacia, aes(x = CIRC, y = AXIS1)) +
   geom_point() +
   geom_smooth(method = "lm") +
-  facet_wrap(~species_id)
+  facet_wrap(~ANT)
 ```
 
 * Combining different data sources
 
 ```
-ggplot(dm_ts_data, aes(x = year, y = count)) +
-  geom_line() +
-  geom_line(data = do_ts_data, aes(x = year, y = count), 
-	color = "red")
+ggplot(acacia, aes(x = CIRC, y = AXIS1)) +
+  geom_point() +
+  geom_point(data = acacia, aes(x = year, y = AXIS2), color = "red") +
+  labs(x = "Circumference [cm]", y = "Canopy Width [m]")
+```
+
+* Each layer will default to `ggplot()` mappings unless modified
+    * So, we don't have to specify the arguments that are the same
+
+```
+ggplot(acacia, aes(x = CIRC, y = AXIS1)) +
+  geom_point() +
+  geom_point(aes(y = AXIS2), color = "red") +
+  labs(x = "Circumference [cm]", y = "Canopy Width [m]")
 ```
 
 > Do Task 5 in [Exercise 3 - Adult vs Newborn Size]({{ site.baseurl }}/exercises/Graphing-adult-vs-newborn-size-R).
 
 ### Statistical transformations
 
-* `geoms` include statistical transformations
+* Geoms include statistical transformations
 * So far we've seen
-    * `identity`: the raw form of the data or no transoformation
-    * `smooth`: model line (e.g., loess, lm)
+    * `identity`: the raw form of the data or no transformation
+    * `smooth`: model line (e.g., `loess`, `lm`)
 * Transformations also exist to make things like histograms, bar plots, etc.
-* Occur as defaults in associated `geoms`
+* Occur as defaults in associated Geoms
 
-* To look at the abundances of different species in the dataset use a bar plot
+* To look at the abundances of different ant associations in the dataset, use a
+bar plot
 
 ```
-surveys <- read.csv("surveys.csv")
-surveys <- filter(surveys, weight > 0)
-ggplot(surveys, aes(x = species_id))
-  + geom_bar()
+ggplot(acacia, aes(x = ANT)) + 
+  geom_bar()
 ```
 
 * Uses the transformation `count` to count the number of rows for each species
 
-* To look at the distribution of weights in the dataset use a histogram
+* To look at the distribution of circumferences in the dataset use a histogram
 
 ```
-ggplot(surveys, aes(x = weight)) +
+ggplot(acacia, aes(x = CIRC)) +
   geom_histogram()
 ```
 
-* Uses `bin` transform: split weights into bins and count rows in each bin
-* These can be combined with all of other things we've learned
+* Uses `bins` argument
```

Comment:
There are two different things going on here that are getting mixed together. First, `geom_histogram()` is using the `bin` statistical transformation. Second, there is a `bins` argument that allows you to set the number of bins used by the `bin` transformation. So, I think this first line should be "Uses `bin` transformation" (or transform) and then the bullet point below can be used to mention the `bins` argument.